### PR TITLE
style: Add Support for Hindi Language in the Android Field Officer App

### PR DESCRIPTION
**Pull request by AbhilashG97**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

**Issue Reference :**

#703, #704  

**After making the changes-**

screenshot_20170630-203331

### CODE REVIEW

**Comment by AbhilashG97** [comment]
@therajanmaurya, I have completed the translations, please let me know if I have to make any further changes. I'd be glad to make the changes.

File: mifosng-android/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -0,0 +1,544 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">मीफोसXड्रायड</string>
```
Comment (therajanmaurya) [inline mifosng-android/src/main/res/values-hi/strings.xml]: No, don't change mifosxdroid into hindi. Stay with english name

Comment (AbhilashG97) [inline mifosng-android/src/main/res/values-hi/strings.xml]: @therajanmaurya, I have made the required changes. Please let me know if I have to make any further changes, I'll be happy to do the needful.

**Comment by therajanmaurya** [comment]
@AbhilashG97 can you please fix the conflicts then I will review and merge

**Comment by AbhilashG97** [comment]
@therajanmaurya, I have fixed the conflicts, please let me know if I have to make any further changes.

**Pull request merged**